### PR TITLE
Added gradle support in UI for spring framework

### PR DESCRIPTION
**Pull request by vamshi-ra**

### CODE REVIEW

**Review by rohith-wedaa — APPROVED** [review]

File: src/pages/Designer/Designer.js [hunk 1]
```diff
@@ -127,13 +127,14 @@ const Designer = ({ update, viewMode = false, sharedMetadata = undefined }) => {
...
     const [credits, setCredits] = useState(0);
-    const [userCredits,setUserCredits]=useState(0);
+    const [userCredits, setUserCredits] = useState(0);
     const [aiServices, setAiServices] = useState([]);
     const reactFlowWrapper = useRef(null);
     const edgeUpdateSuccessful = useRef(true);
     const toastIdRef = useRef();
     const ref = useRef(null);
     const [spinner, setSpinner] = useState(false);
+    const [buildTool, setBuildTool] = useState('maven');
```
Comment (harsha-wedaa) [inline src/pages/Designer/Designer.js:137]: Not Required as the build tool is being set at the application data level.

Comment (vamshi-ra) [reply src/pages/Designer/Designer.js]: Removed the parameter

**Review by harsha-wedaa — COMMENTED** [review]
image

can we show the build tool as a config property for the spring applications here ?

**Comment by rohith-wedaa** [comment]
@harsha-wedaa please review again and merge if it looks good to you.

**Review by harsha-wedaa — APPROVED** [review]
OK

**Pull request merged**